containers/source-build/0001-cleanup-directory-with-all-rpms-which-isn-t-used-any.patch

```diff
@@ -0,0 +1,28 @@
+From 9d66f316158bc84d23a64dfae0a4481cb3a14227 Mon Sep 17 00:00:00 2001
```

Comment:
Is this file got added by mistake?

Reply:
No. If you look at the Dockerfile, you see that this patch is applied to `BuildSourceImage.sh`.